apps/webapp/app/send/ibc-form.tsx

```diff
@@ -0,0 +1,81 @@
+import { useStore } from '../../state';
+import { ibcSelector } from '../../state/ibc';
+import { chains } from './constants';
+import { FilledImage, InputToken } from '../../shared';
+import { validateAmount } from '../../utils';
+import { Button, Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from 'ui';
+
+export const IbcForm = () => {
+  const { amount, asset, chain, validationErrors, setAmount, setAsset, setChain } =
+    useStore(ibcSelector);
+
+  return (
+    <form
+      className='flex flex-col gap-2'
+      onSubmit={e => {
+        e.preventDefault();
+      }}
+    >
+      <InputToken
+        label='Amount to send'
+        placeholder='Enter an amount'
+        className='mb-1'
+        asset={asset}
+        setAsset={setAsset}
+        value={amount}
+        onChange={e => {
+          if (Number(e.target.value) < 0) return;
+          setAmount(e.target.value);
+        }}
+        validations={
+          // if the user has no assets, do not confirm the validation
+          asset
+            ? [
+                {
+                  type: 'error',
+                  issue: 'insufficient funds',
+                  checkFn: (amount: string) => validateAmount(amount, asset.balance),
+                },
+              ]
+            : undefined
+        }
+      />
+      <div className='flex flex-col gap-3 rounded-lg border bg-background px-4 pb-5 pt-3'>
+        <p className='text-base font-bold'>Chain</p>
+        <Select
+          value={chain?.name ?? ''}
+          onValueChange={e => setChain(chains.find(i => i.name === e))}
+        >
```

Comment:
<img width="410" alt="Screenshot 2023-10-06 at 10 16 28 AM" src="https://github.com/penumbra-zone/web/assets/16624263/96728df2-c68a-4fa1-832c-6216896fb921">

We need to revamp this design. 
- It kinda blends in with the item. Perhaps it should be a dark gray.
- On hover, the background of that row should be highlighted
- I don't think it's wide enough. It should be the width of the thing being selected.
- If the item is already selected, we can just remove it from the drop down list (at the moment we have that check)

Reply:
I am waiting for the ui kit from Daria